ql/src/test/results/clientpositive/tez/tez_union_udtf.q.out

```diff
@@ -140,7 +137,7 @@ POSTHOOK: Input: default@src
 POSTHOOK: Input: default@src1
 POSTHOOK: Output: database:default
 POSTHOOK: Output: default@x
-POSTHOOK: Lineage: x.key EXPRESSION [(src)src.FieldSchema(name:key, type:string, comment:default), (src1)src1.FieldSchema(name:key, type:string, comment:default), ]
```

Comment:
This seems lineage change seems like a regression that we may need to look into. Did we lose the information that the "key" column comes from the `src` table?